### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by meta-sabir-mohammad**
15-07-2019

### CODE REVIEW

File: Assignment2A/ArithmeticOperation.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import java.io.IOException;
+
+/**
+ * This Interface provide generic method for arithmetic operation  
+ *
+ */
+public interface ArithmeticOperation {
+	
+	/**
+	 *
```
Comment (ghost) [inline Assignment2A/ArithmeticOperation.java:12]: Please try to add purpose of method in comment and follow the order of comment like purpose then param then return detail of method in sequence.

File: Assignment2B/JobScheduler.java [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+	/**
+	 * @param process take two dimension array which contain arrival time and burst time of n process
+	 * @param numberOfProcess take number of process in Integer
+	 * @return an one dimension array which contain waiting time for all process
+	 */
+	public int[] getWaitingTime(int process[][], int numberOfProcess) throws IOException {
+		
+		//input validation
+		boolean validInput = inputValidation(process,numberOfProcess);
+		if(validInput == false){
```
Comment (ghost) [inline Assignment2B/JobScheduler.java:23]: Use boolean input directly in condition like if(validInput) or if(!validInput) .

File: Assignment2A/ArithmeticOperation.java (hunk 1 above)
Comment (meta-sabir-mohammad) [reply Assignment2A/ArithmeticOperation.java:12]: ok sir

File: Assignment2B/JobScheduler.java (hunk 2 above)
Comment (meta-sabir-mohammad) [reply Assignment2B/JobScheduler.java:23]: i will from now on

**Pull request closed without merging**